allink_core/core_apps/allink_teaser_grid/cms_plugins.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+from cms.models import CMSPlugin
+from cms.plugin_base import CMSPluginBase
+from cms.plugin_pool import plugin_pool
+
+from django import forms
+from django.conf import settings
+from allink_core.core.utils import get_additional_choices
+
+from .models import AllinkTeaserGridContainerPlugin
+
+
+class AllinkTeaserGridContainerPluginForm(forms.ModelForm):
+    class Meta:
+        model = AllinkTeaserGridContainerPlugin
+        exclude = ('page', 'position', 'placeholder', 'language', 'plugin_type')
+
+    def __init__(self, *args, **kwargs):
+        super(AllinkTeaserGridContainerPluginForm, self).__init__(*args, **kwargs)
+
+        if get_additional_choices('CONTENT_CSS_CLASSES'):
```

Comment:
do you think it would make sense to use new settings vars for all these new plugins. e.g 
 TEASER_GRID_CONTAINER_CSS_CLASSES ... or do you think they will share the same classes? 

should we at least use other classes as the CONTENT plugin?